guides/common/attributes-base.adoc

```diff
@@ -19,6 +19,8 @@
 :TuningDocURL: {BaseURL}Tuning_Performance/{BaseFilenameURL}#
 :UpdatingDocURL: {BaseURL}Updating_Project/{BaseFilenameURL}#
 :UpgradingDocURL: {BaseURL}Upgrading_Project/{BaseFilenameURL}#
+:UpgradingConnectedDocURL: {BaseURL}Upgrading_Connected_Project/{BaseFilenameURL}#
```

Comment:
May I suggest keeping the `Upgrading_Project` naming (without Connected) for the connected solution that is the only supported solution in upstream? This also applies to the `UpgradingDoc*` attributes and `doc-Upgrading_Project` folder. It doesn't affect downstream.
I'm suggesting this, because that's how it's currently done for the Installing guides.

The advantage of this solution, apart from consistency, is that you will not have to update links to the new title.

Is it clear what I'm suggesting?

Reply:
Ack @Lennonka , I'll tweak the Upgrading_Project.

Reply:
@Lennonka I haven't deleted the attributes yet. If the changes in the latest commit look okay, I'll go ahead and delete the attributes and files for the connected guide that I created.

Reply:
@AkshayGadhaveRH If you have used the same contents for doc-Upgrading_Project/master.adoc as you did in doc-Upgradinc_Connected_Project/master.adoc, you should be fine. You can go ahead and remove the unnecessary files/attributes (keep the change in a single commit, just to make sure).

Reply:
done